### ISSUE TITLE
lmql questions (task 2aa & 2b)

### ISSUE THREAD

**Issue by nliyanage** [mikeizbicki/modulus-magnus-linguae#8]
hi all! 

iraj and i are working on tasks 2aa & 2b, and we were wondering what the deliverable should be for 2b. we have code that will create the text that needs to be pasted into the lmql playground, but we were wondering if we are meant to find a way to run this on the lambda server instead of the lmql playground website. 

for reference, we created a json file w sample questions & answers, and python code to read that file. 

any tips on how to figure out how to run it on the lambda server? or are we approaching this wrong? is there a different deliverable we should aim for? when we type "lmql run" into the terminal, we get "lmql: command not found" error

**Comment by mikeizbicki** [comment]
Yes, you should try to get an automatic system setup that will process your lmql script without any manual intervention.

`lmql` is a python program installed with `pip`.  So if you haven't installed it then that's why you are getting an error when trying to run it.  You will have to give it an openai api key to get it to work.  For now, you can use a free trial api key.  When we start doing more intensive processing that will exceed the free tier, we'll get you connected to my account to pay for the compute.

**Comment by irajmoradi** [comment]
I am having an error when trying to install lmql through pip. I am getting the below messages.

```
pip3 install lmql
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement lmql (from versions: none)
ERROR: No matching distribution found for lmql
```

I have also tried just pip instead of pip3, as well as python3 -m pip lmql, with the same errors appearing.

**Comment by mikeizbicki** [comment]
I believe this is caused by the old version of python and will be fixed by following the instructions in #10.
